CSharp/AccountManagement/packages.config

```diff
@@ -3,12 +3,14 @@
   <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Batch" version="1.5.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Resources" version="2.18.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Batch" version="2.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.1.1-preview" targetFramework="net45" />
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />

```

Comment:
Same, do we still need the Bcl packages and the Hyak.Common package now that we've moved to the AutoRest proxies?